### ISSUE TITLE
[question] SWA BatchNorm momentum reset

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/contrib#29]
Could you please comment on the particular way of decaying the BatchNorm momentum parameter for every mini-batch during BatchNorm parameters update:
https://github.com/pytorch/contrib/blob/master/torchcontrib/optim/swa.py#L305

(As far as I understand, BatchNorm momentum is usually constant)

Thanks!
